### PR TITLE
Fix savings on android

### PR DESCRIPTION
**Pull request by brunobar79**
A hack on top of @osdnk hack to get the ripple effect working without breaking the onPress behavior

PoW: http://recordit.co/nhFgwONFaP

### CODE REVIEW

**Review by osdnk — APPROVED** [review]
o, I just came across this!

**Comment by osdnk** [comment]
or wait

**Review by osdnk — CHANGES_REQUESTED** [review]
image
please just alter this line

https://github.com/rainbow-me/rainbow/pull/1332/files#diff-c67c188b0dd37aeda3c32a1770e4a9308986b74561e6230e435bf60e51b1bf62R172

**Comment by brunobar79** [comment]
@osdnk  This one is ready 2 go

File: src/components/savings/SavingsListRow.js [hunk 1]
```diff
@@ -166,7 +166,7 @@ const SavingsListRow = ({
             as={android && ButtonPressAnimation}
             css={padding(9, 10, 10, 11)}
             justify="space-between"
-            onPress={NOOP}
+            onPress={android ? onButtonPress : NOOP}
```
Comment (osdnk) [inline src/components/savings/SavingsListRow.js:169]:
```suggestion
            onPress={onButtonPress}
```

I this it's sufficient, cause `Row` does not have onPress

**Review by osdnk — APPROVED** [review]
nit

**Pull request merged**